File: src/Java/myservicebus-abstractions/src/main/java/com/myservicebus/MessageHeaders.java
package com.myservicebus;

public final class MessageHeaders {
    private MessageHeaders() {}

    public static final String FAULT_ADDRESS = "MT-Fault-Address";

    public static final String EXCEPTION_TYPE = "MT-ExceptionType";
    public static final String EXCEPTION_MESSAGE = "MT-ExceptionMessage";
    public static final String EXCEPTION_STACKTRACE = "MT-ExceptionStackTrace";
    public static final String REASON = "MT-Reason";
    public static final String REDELIVERY_COUNT = "MT-RedeliveryCount";

    public static final String HOST_MACHINE = "MT-Host-MachineName";
    public static final String HOST_PROCESS = "MT-Host-ProcessName";
    public static final String HOST_PROCESS_ID = "MT-Host-ProcessId";
    public static final String HOST_ASSEMBLY = "MT-Host-Assembly";
    public static final String HOST_ASSEMBLY_VERSION = "MT-Host-AssemblyVersion";
    public static final String HOST_FRAMEWORK_VERSION = "MT-Host-FrameworkVersion";
    public static final String HOST_MASS_TRANSIT_VERSION = "MT-Host-MassTransitVersion";
    public static final String HOST_OS_VERSION = "MT-Host-OperatingSystemVersion";
}

File: src/MyServiceBus.Abstractions/MessageHeaders.cs
namespace MyServiceBus;

public static class MessageHeaders
{
    public const string FaultAddress = "MT-Fault-Address";

    public const string ExceptionType = "MT-ExceptionType";
    public const string ExceptionMessage = "MT-ExceptionMessage";
    public const string ExceptionStackTrace = "MT-ExceptionStackTrace";
    public const string Reason = "MT-Reason";
    public const string RedeliveryCount = "MT-RedeliveryCount";

    public const string HostMachineName = "MT-Host-MachineName";
    public const string HostProcessName = "MT-Host-ProcessName";
    public const string HostProcessId = "MT-Host-ProcessId";
    public const string HostAssembly = "MT-Host-Assembly";
    public const string HostAssemblyVersion = "MT-Host-AssemblyVersion";
    public const string HostFrameworkVersion = "MT-Host-FrameworkVersion";
    public const string HostMassTransitVersion = "MT-Host-MassTransitVersion";
    public const string HostOperatingSystemVersion = "MT-Host-OperatingSystemVersion";
}

